_projects/2024/100540514.Rmd

```diff
@@ -0,0 +1,554 @@
+install.packages("distill")
+
+library(distill)
+
+---
+title: "Dispersion patterns of SARS-CoV-2 variants Gamma, Lambda and Mu in Latin America and the Caribbean"
+description: |
+The graph illustrates the total number of genomes sequenced alongside the proportion of sequenced COVID-19 cases.
+categories: "2025"
+author: Gina Tedesco
+date: "`r Sys.Date()`"
+output:
+  distill::distill_article:
+  self_contained: FALSE
+  toc: TRUE
+
+---
+
+#The COVID-19 pandemic disproportionately impacted Latin America and the Caribbean (LAC), revealing significant inequalities in health infrastructure and surveillance capacity across the region. In response, the Pan American Health Organization (PAHO) established the COVID-19 Genomic Surveillance Regional Network (COVIGEN) to strengthen genomic sequencing and enable the timely identification of viral variants. From February 2020 to March 2022, this initiative resulted in the sequencing of 126,985 SARS-CoV-2 genomes across 32 LAC countries and territories. Expanding this dataset with genomes from the GISAID database brought the total analyzed to 296,286 genomes from 41 regions, providing a comprehensive view of genomic diversity. The study highlighted disparities in sequencing efforts, ranging from below 0.05% to over 10% of total COVID-19 cases, with countries such as Brazil, Argentina, Colombia, Mexico, and Peru contributing 80% of the sequences. The emergence of regional Variants of Interest (VOIs) like Gamma, Lambda, and Mu emphasized the importance of COVIGEN in tracking viral evolution and guiding public health responses. This analysis further explored the dynamics of variant dissemination by incorporating air traffic data and mobility restrictions to model viral spread patterns, offering insights into the interplay between epidemiology and social behavior during the pandemic.
+
+![](images/Orig Graph.png){width="100%"}
```

Comment:
`images/Orig Graph.png` is not part of the pull request! :) Please commit and push it too.